src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.cpp

```diff
@@ -579,14 +579,9 @@ void ShenandoahHeapRegion::recycle() {
 
   set_top(bottom());
   clear_live_data();
-  heap->marking_context()->clear_bitmap(this);
```

Comment:
This is a merge error on this branch. `shenandoah-jdk21u` does not reset the bitmaps twice like this.

Reply:
yikes :-)